pkgs/development/python-modules/opuslib/default.nix

```
@@ -0,0 +1,36 @@
+{
+  buildPythonPackage,
```

most packages will start on the first line
```suggestion
{ buildPythonPackage,
```